content/en/docs/ambient/architecture/index.md

```diff
@@ -33,9 +84,8 @@ However, depending on the destination's capabilities, different behavior will oc
 If the destination is also captured, or otherwise has Istio proxy capabilities (such as a sidecar), the request will be upgraded to an encrypted {{< gloss "HBONE" >}}HBONE tunnel{{< /gloss >}}.
 If the destination has a waypoint proxy, in addition to being upgraded to HBONE, the request will instead be forwarded to that waypoint.
 
-Note that in the case of a request to a `Service`, a specific endpoint will be selected to determine if it has a waypoint.
-However, if it *has* a waypoint, the request will be sent with a target destination of the `Service`, not the selected endpoint.
-This allows the waypoint to apply service-oriented policies to the traffic.
+Note that in the case of a request to a `Service`, if the service *has* a waypoint and the waypoint can capture `service` or `all` traffic, the request will be sent to its waypoint to enforce service-oriented policies to the traffic.
```

Comment:
I don't think we need to keep qualifying that the waypoint is correctly configured to habdle 'service' or 'all' traffic. We've covered it above so we can elide this here.


Also I would s/service-oriented policies/L7 policies  here and elsewhere

Same for 'pod oriented' below, I think these terms are likely to confuse

Reply:
updated